### ISSUE TITLE
Create haplotype caller reassembly plugin for IGV

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/gatk#3286]
Create a plugin for IGV that allows it to display the reads as haplotype caller has assembled them.  This would be extremely useful for analysts who are doing manual review and would mostly obviate the need for bamOut.

Jim Robinson seems interested in this so we may be able to get help from IGV to do the integration https://github.com/igvteam/igv/issues/428.

**Comment by tfenne** [comment]
Yes please!  I'd also find this incredibly useful.  @lbergelson any thoughts on how hard it would be to generate a separate build artifact for the local reassembly code that didn't have too many dependencies?

**Comment by jrobinso** [comment]
Hey all,  I'm still interested in supporting this.   We don't really have a "plugin API",  I am in fact the API, but if you give me something usable I'll plug it in.   As this is marked "QuixoticDream" I don't think that's likely.   I'm closing the corresponding IGV issue,  too many open issues, but it doesn't mean I've lost interest.

**Comment by lbergelson** [comment]
I would still love to do this.  I haven't had a chance to work on it though.
